include/osquery/sql.h

```diff
@@ -127,6 +127,12 @@ class SQL : private only_movable {
                                  ConstraintOperator op,
                                  const std::string& expr);
 
+  static QueryData selectFrom(const std::initializer_list<std::string> columns,
```

Comment:
nit: Would you mind adding some comments for this function similar to those above it? Doesn't have to be super verbose as they're so closely related, but for consistency